packages/EasyErrorHandler/src/Bridge/Symfony/DependencyInjection/EasyErrorHandlerExtension.php

```diff
@@ -25,6 +25,8 @@ public function load(array $configs, ContainerBuilder $container): void
         $config = $this->processConfiguration(new Configuration(), $configs);
         $loader = new PhpFileLoader($container, new FileLocator([__DIR__ . '/../Resources/config']));
 
+        $container->setParameter(BridgeConstantsInterface::PARAM_IS_API_PLATFORM, $config['api_platform']);
```

Comment:
@itorgov here same, let's make sure we update the constant/parameter accordingly to the new config name.

Reply:
Done.